src/mds/cephfs_features.h

```diff
@@ -33,6 +33,7 @@
 #define CEPHFS_FEATURE_MULTI_RECONNECT  12
 /* N.B.: oops, we can't use CEPHFS_FEATURE_NAUTILUS because we didn't add a bit for it. */
```

Comment:
maybe:

#define CEPHFS_FEATURE_NAUTILUS   CEPHFS_FEATURE_MULTI_RECONNECT

Reply:
My interpretation of the extra bit for a release is that the client has code "up-to-date" with the rest of Ceph, not just the features CephFS cares about. So it's not really appropriate to alias to another feature bit. What do you think?

Reply:
I added CEPHFS_FEATURE_MULTI_RECONNECT support for ceph-fuse and kernel about the same time. when ceph-fuse and kernel client support CEPHFS_FEATURE_MULTI_RECONNECT, they also support CEPHFS_FEATURE_REPLY_ENCODING and CEPHFS_FEATURE_LAZY_CAP_WANTED. So it's good enough to rename CEPHFS_FEATURE_MULTI_RECONNECT to CEPHFS_FEATURE_NAUTILUS

Reply:
Okay, I changed it.